### PR TITLE
docs(746): hide e2e tests from storybook

### LINKED ISSUE #746: Standardise hiding e2e test stories from sidebar
# Description

Recommendation 2 from: 

https://nidigitalsolutions.jira.com/wiki/spaces/NPP/pages/4333765497/PPDSC-2222+Storybook+visual+testing+open+state 

All stories and scenarios that only exist for test purposes should be named so that they are automatically hidden from the sidebar. Test code will still be able to navigate directly to the stories. 

- Add `hidden` to the end of storybook component name(s)
- Add CSS to storybook to hide these stories in the navigation

```
export const StoryImageLazyLoadingHidden = () => {
  // etc
};
StoryImageLazyLoadingHidden.storyName = 'Lazy loading';
```

```
<style>
    [id$='hidden']{
        display: none !important;
    }
</style>
```

# ACs:

No e2e test stories showing in sidebar

Stories still accessible in e2e tests

Image, Audio Player, etc follow the same approach if required.

### PR DESCRIPTION
**Pull request by mutebg**
#746

Closes #746

**What**

1. Background - hide stories which are only for e2e testing
2. What did you do - renamed stories as adding suffix "hidden" in that way we can target these stories with CSS selector so that they are hidden.
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/746-hide-e2e-tests-from-storybook
  Storybook: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/746-hide-e2e-tests-from-storybook/storybook

File: src/image/__tests__/image-e2e.stories.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ const ImageContainer = styled.div`
 `;
 
 export default {
-  title: 'Components/image-e2e',
+  title: 'Components/image-e2e-hidden',
   component: () => 'None',
 };
```
Comment (JohnTParsons) [inline src/image/__tests__/image-e2e.stories.tsx:13]: Should StoryE2eTest also be renamed StoryE2eTestHidden, as per the example on the ticket? Is a bit confusing

File: src/audio-player-composable/__tests__/audio-player-composable-e2e.stories.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ import {
 import {calculateTime} from '../components/time-display/utils';
 
 export default {
-  title: 'Components/audio-player-composable-e2e',
+  title: 'Components/audio-player-composable-e2e-hidden',
   component: () => 'None',
 };
```
Comment (JohnTParsons) [inline src/audio-player-composable/__tests__/audio-player-composable-e2e.stories.tsx:22]: Should AudioPlayerE2E also be renamed AudioPlayerE2EHidden, as per the example on the ticket? Is a bit confusing

File: src/image/__tests__/image-e2e.stories.tsx (hunk 1 above)
Comment (mutebg) [reply src/image/__tests__/image-e2e.stories.tsx:13]: Its not needed, this file has only 1 scenario, and we want to hide the whole file, not only the scenario, in that case hidden is added to the title. Same applies to audio-player comment

**Review by JohnTParsons — APPROVED** [review]

**Review by Vanals — APPROVED** [review]

**Pull request merged**